### PR TITLE
Provide reason for coins being granted

### PR DESCRIPTION
**Pull request by jedrzej**

### CODE REVIEW

**Comment by jacekelgda** [comment]
@jedrzej could you please add test for `extractMentionedUsersAndReason` ?

**Comment by jedrzej** [comment]
@jacekelgda done

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,6 @@ serverless.env.yml
 
 # Build
 build
+
+# Other
+.idea
```
Comment (jacekelgda) [inline .gitignore]: please add new line

**Comment by jacekelgda** [comment]
@jedrzej also pls update your branch with master changes

**Comment by jacekelgda** [comment]
@ryanchartrand can you merge ?

**Pull request merged**